src/popover/popover.ts

```diff
@@ -64,26 +65,41 @@ export class NgbPopover implements OnInit, OnDestroy {
    */
   @Input() container: string;
   /**
+   * When set to a truthy value, closes the popover when a click occurs outside of the popover element
+   */
+  @Input() closeOnClickOutside: boolean;
+  /**
    * Emits an event when the popover is shown
    */
   @Output() shown = new EventEmitter();
   /**
    * Emits an event when the popover is hidden
    */
   @Output() hidden = new EventEmitter();
+  /**
+   * Emits an event when a click occurs outside of the popover element.
+   */
+  @Output() clickOutside = new EventEmitter();
```

Comment:
The popover wouldn't necessarily be closed when a user clicks outside of it; that's just how I have it set up in the demo, and was the idea behind creating it.

I've actually been thinking that (clickOutside) should be a more generic directive not specific to the popover. I'm not sure if it would be useful for other NGB components, though.

I'm OK with getting rid of it. I just thought it would be a more flexible solution than [closeOnClickOutside].